web/.env

```diff
@@ -1,3 +1,3 @@
 VUE_APP_API_URL=http://localhost:3000
 # Enable the line below to develop for server mode
-# VUE_APP_MODE=server
+VUE_APP_MODE=server

```

Comment:
I have added a shortcut for developing for the server: `yarn serve:server` and `yarn build:server`.